Support for sql type DATE

Hi,  
I will be great to have the support of the sql type `DATE` in the library.  
I imagine this one will be done with `chrono::NaiveDate` since it's the one that is used in the [documentation](https://www.sea-ql.org/SeaORM/docs/generate-entity/entity-structure#column-type). 

To give my own example, I have a model `Rate` which have a simple date:

```rust
use chrono::NaiveDate;
use sea_orm::entity::prelude::*;
use sea_orm::DeriveEntityModel;

#[derive(Debug, Clone, PartialEq, DeriveEntityModel)]
#[sea_orm(table_name = "currency_exchange_rate")]
pub struct Model {
    #[sea_orm(primary_key)]
    pub currency_exchange_rate_id: i64,
    pub from_currency: String,
    pub to_currency: String,
    pub rate_date: NaiveDate,
    pub rate: f64,
}

#[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
pub enum Relation {}

impl ActiveModelBehavior for ActiveModel {}
```

But this implementation give me a compile error:

```
error[E0277]: the trait bound `NaiveDate: TryGetable` is not satisfied
  --> src/rates/mod.rs:11:35
   |
11 | #[derive(Debug, Clone, PartialEq, DeriveEntityModel)]
   |                                   ^^^^^^^^^^^^^^^^^ the trait `TryGetable` is not implemented for `NaiveDate`
   |
   = note: this error originates in the derive macro `DeriveEntityModel` (in Nightly builds, run with -Z macro-backtrace for more info)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -34,7 +34,7 @@ sea-query = { version = "^0.16.3", features = ["thread-safe"] }
 sea-strum = { version = "^0.21", features = ["derive", "sea-orm"] }
 serde = { version = "^1.0", features = ["derive"] }
 serde_json = { version = "^1", optional = true }
-sqlx = { version = "^0.5", optional = true }
+sqlx = { version = "^0.5", git = "https://github.com/billy1624/sqlx.git", branch = "fix-sqlite-naive-time", optional = true }
 uuid = { version = "0.8", features = ["serde", "v4"], optional = true }
 
 [dev-dependencies]

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,7 +30,7 @@ futures-util = { version = "^0.3" }
 log = { version = "^0.4", optional = true }
 rust_decimal = { version = "^1", optional = true }
 sea-orm-macros = { version = "^0.2.3", path = "sea-orm-macros", optional = true }
-sea-query = { version = "^0.16.4", git = "https://github.com/SeaQL/sea-query.git", branch = "sqlite-time-temp-fixup", features = ["thread-safe"] }
+sea-query = { version = "^0.16.5", features = ["thread-safe"] }
 sea-strum = { version = "^0.21", features = ["derive", "sea-orm"] }
 serde = { version = "^1.0", features = ["derive"] }
 serde_json = { version = "^1", optional = true }

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,11 +30,11 @@ futures-util = { version = "^0.3" }
 log = { version = "^0.4", optional = true }
 rust_decimal = { version = "^1", optional = true }
 sea-orm-macros = { version = "^0.2.3", path = "sea-orm-macros", optional = true }
-sea-query = { version = "^0.16.3", features = ["thread-safe"] }
+sea-query = { version = "^0.16.4", git = "https://github.com/SeaQL/sea-query.git", branch = "sqlite-time-temp-fixup", features = ["thread-safe"] }
 sea-strum = { version = "^0.21", features = ["derive", "sea-orm"] }
 serde = { version = "^1.0", features = ["derive"] }
 serde_json = { version = "^1", optional = true }
-sqlx = { version = "^0.5", git = "https://github.com/billy1624/sqlx.git", branch = "fix-sqlite-naive-time", optional = true }
+sqlx = { version = "^0.5", optional = true }
 uuid = { version = "0.8", features = ["serde", "v4"], optional = true }
 
 [dev-dependencies]

--- a/sea-orm-macros/src/derives/entity_model.rs
+++ b/sea-orm-macros/src/derives/entity_model.rs
@@ -171,7 +171,7 @@ pub fn expand_derive_entity_model(data: Data, attrs: Vec<Attribute>) -> syn::Res
                                 "f64" => quote! { Double },
                                 "bool" => quote! { Boolean },
                                 "Date" | "NaiveDate" => quote! { Date },
-                                "NaiveTime" => quote! { Time },
+                                "Time" | "NaiveTime" => quote! { Time },
                                 "DateTime" | "NaiveDateTime" => {
                                     quote! { DateTime }
                                 }

--- a/sea-orm-macros/src/derives/entity_model.rs
+++ b/sea-orm-macros/src/derives/entity_model.rs
@@ -170,7 +170,7 @@ pub fn expand_derive_entity_model(data: Data, attrs: Vec<Attribute>) -> syn::Res
                                 "f32" => quote! { Float },
                                 "f64" => quote! { Double },
                                 "bool" => quote! { Boolean },
-                                "NaiveDate" => quote! { Date },
+                                "Date" | "NaiveDate" => quote! { Date },
                                 "NaiveTime" => quote! { Time },
                                 "DateTime" | "NaiveDateTime" => {
                                     quote! { DateTime }

--- a/src/entity/prelude.rs
+++ b/src/entity/prelude.rs
@@ -13,6 +13,9 @@ pub use serde_json::Value as Json;
 pub use chrono::NaiveDate as Date;
 
 #[cfg(feature = "with-chrono")]
+pub use chrono::NaiveTime as Time;
+
+#[cfg(feature = "with-chrono")]
 pub use chrono::NaiveDateTime as DateTime;
 
 #[cfg(feature = "with-chrono")]

--- a/src/entity/prelude.rs
+++ b/src/entity/prelude.rs
@@ -10,6 +10,9 @@ pub use crate::{
 pub use serde_json::Value as Json;
 
 #[cfg(feature = "with-chrono")]
+pub use chrono::NaiveDate as Date;
+
+#[cfg(feature = "with-chrono")]
 pub use chrono::NaiveDateTime as DateTime;
 
 #[cfg(feature = "with-chrono")]

--- a/src/executor/query.rs
+++ b/src/executor/query.rs
@@ -242,6 +242,9 @@ try_getable_all!(Vec<u8>);
 try_getable_all!(serde_json::Value);
 
 #[cfg(feature = "with-chrono")]
+try_getable_all!(chrono::NaiveDate);
+
+#[cfg(feature = "with-chrono")]
 try_getable_all!(chrono::NaiveDateTime);
 
 #[cfg(feature = "with-chrono")]

--- a/src/executor/query.rs
+++ b/src/executor/query.rs
@@ -245,6 +245,9 @@ try_getable_all!(serde_json::Value);
 try_getable_all!(chrono::NaiveDate);
 
 #[cfg(feature = "with-chrono")]
+try_getable_all!(chrono::NaiveTime);
+
+#[cfg(feature = "with-chrono")]
 try_getable_all!(chrono::NaiveDateTime);
 
 #[cfg(feature = "with-chrono")]

--- a/tests/common/bakery_chain/metadata.rs
+++ b/tests/common/bakery_chain/metadata.rs
@@ -9,6 +9,7 @@ pub struct Model {
     pub value: String,
     pub bytes: Vec<u8>,
     pub date: Date,
+    pub time: Time,
 }
 
 #[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]

--- a/tests/common/bakery_chain/metadata.rs
+++ b/tests/common/bakery_chain/metadata.rs
@@ -8,6 +8,7 @@ pub struct Model {
     pub key: String,
     pub value: String,
     pub bytes: Vec<u8>,
+    pub date: Date,
 }
 
 #[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]

--- a/tests/common/setup/schema.rs
+++ b/tests/common/setup/schema.rs
@@ -287,6 +287,7 @@ pub async fn create_metadata_table(db: &DbConn) -> Result<ExecResult, DbErr> {
         .col(ColumnDef::new(metadata::Column::Value).string().not_null())
         .col(ColumnDef::new(metadata::Column::Bytes).binary().not_null())
         .col(ColumnDef::new(metadata::Column::Date).date().not_null())
+        .col(ColumnDef::new(metadata::Column::Time).time().not_null())
         .to_owned();
 
     create_table(db, &stmt, Metadata).await

--- a/tests/common/setup/schema.rs
+++ b/tests/common/setup/schema.rs
@@ -286,6 +286,7 @@ pub async fn create_metadata_table(db: &DbConn) -> Result<ExecResult, DbErr> {
         .col(ColumnDef::new(metadata::Column::Key).string().not_null())
         .col(ColumnDef::new(metadata::Column::Value).string().not_null())
         .col(ColumnDef::new(metadata::Column::Bytes).binary().not_null())
+        .col(ColumnDef::new(metadata::Column::Date).date().not_null())
         .to_owned();
 
     create_table(db, &stmt, Metadata).await

--- a/tests/parallel_tests.rs
+++ b/tests/parallel_tests.rs
@@ -26,6 +26,7 @@ pub async fn crud_in_parallel(db: &DatabaseConnection) -> Result<(), DbErr> {
             value: "1.18".to_owned(),
             bytes: vec![1, 2, 3],
             date: Date::from_ymd(2021, 9, 27),
+            time: Time::from_hms(11, 32, 55),
         },
         metadata::Model {
             uuid: Uuid::new_v4(),
@@ -33,6 +34,7 @@ pub async fn crud_in_parallel(db: &DatabaseConnection) -> Result<(), DbErr> {
             value: "0.78".to_owned(),
             bytes: vec![1, 2, 3],
             date: Date::from_ymd(2021, 9, 27),
+            time: Time::from_hms(11, 32, 55),
         },
         metadata::Model {
             uuid: Uuid::new_v4(),
@@ -40,6 +42,7 @@ pub async fn crud_in_parallel(db: &DatabaseConnection) -> Result<(), DbErr> {
             value: "1.1".to_owned(),
             bytes: vec![1, 2, 3],
             date: Date::from_ymd(2021, 9, 27),
+            time: Time::from_hms(11, 32, 55),
         },
     ];
 

--- a/tests/parallel_tests.rs
+++ b/tests/parallel_tests.rs
@@ -25,18 +25,21 @@ pub async fn crud_in_parallel(db: &DatabaseConnection) -> Result<(), DbErr> {
             key: "markup".to_owned(),
             value: "1.18".to_owned(),
             bytes: vec![1, 2, 3],
+            date: Date::from_ymd(2021, 9, 27),
         },
         metadata::Model {
             uuid: Uuid::new_v4(),
             key: "exchange_rate".to_owned(),
             value: "0.78".to_owned(),
             bytes: vec![1, 2, 3],
+            date: Date::from_ymd(2021, 9, 27),
         },
         metadata::Model {
             uuid: Uuid::new_v4(),
             key: "service_charge".to_owned(),
             value: "1.1".to_owned(),
             bytes: vec![1, 2, 3],
+            date: Date::from_ymd(2021, 9, 27),
         },
     ];
 

--- a/tests/uuid_tests.rs
+++ b/tests/uuid_tests.rs
@@ -24,6 +24,7 @@ pub async fn create_metadata(db: &DatabaseConnection) -> Result<(), DbErr> {
         value: "1.18".to_owned(),
         bytes: vec![1, 2, 3],
         date: Date::from_ymd(2021, 9, 27),
+        time: Time::from_hms(11, 32, 55),
     };
 
     let res = Metadata::insert(metadata.clone().into_active_model())

--- a/tests/uuid_tests.rs
+++ b/tests/uuid_tests.rs
@@ -23,6 +23,7 @@ pub async fn create_metadata(db: &DatabaseConnection) -> Result<(), DbErr> {
         key: "markup".to_owned(),
         value: "1.18".to_owned(),
         bytes: vec![1, 2, 3],
+        date: Date::from_ymd(2021, 9, 27),
     };
 
     let res = Metadata::insert(metadata.clone().into_active_model())

